exercises/practice/triangle/triangle.f90

```diff
@@ -0,0 +1,12 @@
+
+module triangle
```

Comment:
We should give stubs for all three functions under test. `equilateral`, `isosceles` and `scalene`